sencha-workspace/packages/slate-cbl/src/view/skill/OverviewBody.js

```diff
@@ -0,0 +1,92 @@
+                        '<tpl for=".">',
+                            '<tr class="skill-grid-demo-row" data-demonstration="{ID}">',
+                                '<td class="skill-grid-demo-data skill-grid-demo-index">{[xindex]}</td>',
+                                '<td class="skill-grid-demo-data skill-grid-demo-date">{Demonstration.Demonstrated:date("M j, Y")}</td>',
+                                '<td class="skill-grid-demo-data skill-grid-demo-level"><div class="level-color cbl-level-{Level}"><tpl if="Level==0">M<tpl else>{Level}</tpl></div></td>',
+                                '<td class="skill-grid-demo-data skill-grid-demo-type">{Demonstration.ExperienceType:htmlEncode}</td>',
+                                '<td class="skill-grid-demo-data skill-grid-demo-context">{Demonstration.Context:htmlEncode}</td>',
+                                '<td class="skill-grid-demo-data skill-grid-demo-task">{Demonstration.PerformanceType:htmlEncode}</td>',
+                                // '<td class="skill-grid-demo-data skill-grid-demo-artifact"><tpl if="Demonstration.ArtifactURL"><a href="{Demonstration.ArtifactURL:htmlEncode}">{Demonstration.ArtifactURL:htmlEncode}</a></tpl></td>',
+                            '</tr>',
+                            '<tr class="skill-grid-demo-detail-row" data-demonstration="{ID}">',
+                                '<td class="skill-grid-demo-detail-data" colspan="6">',
+                                    '<div class="skill-grid-demo-detail-ct">',
+                                        '<tpl if="Demonstration.ArtifactURL">',
+                                            '<div class="skill-grid-demo-artifact">',
+                                                '<strong>Artifact: </strong>',
+                                                '<a href="{Demonstration.ArtifactURL:htmlEncode}" target="_blank">{Demonstration.ArtifactURL:htmlEncode}</a>',
+                                            '</div>',
+                                        '</tpl>',
+                                        '<tpl if="Demonstration.Comments">',
+                                            '<div class="skill-grid-demo-comments">',
+                                                '<strong>Comments: </strong>',
+                                                '{[this.escapeAndNewline(values.Demonstration.Comments)]}',
+                                            '</div>',
+                                        '</tpl>',
+                                        '<div class="skill-grid-demo-meta">',
+                                            'Demonstration #{Demonstration.ID} &middot;&nbsp;',
+                                            '<tpl for="Demonstration.Creator"><a href="/people/{Username}">{FirstName} {LastName}</a></tpl> &middot;&nbsp;',
+                                            '{[fm.date(new Date(values.Demonstration.Created * 1000), "F j, Y, g:i a")]} &middot;&nbsp;',
```

Comment:
Best practice would be to convert it to a JavaScript Date instance at the data layer, not just to milliseconds. Usually this is handled by the fields definition in an Ext.data.Model subclass. This is a rather deeply nested object though and I got lazy with pre-processing it before rendering. For nested data using full-blown Ext.data.Model instances usually isn't worthwhile, but I agree that at least the controller or even better API wrapper method should be looping through and converting values before they get to the rendering layer.